docs/wkt/wkt_mesh.h

```diff
@@ -0,0 +1,28 @@
+#ifndef wkt_mesh_h
+#define wkt_mesh_h
+#include "ir_start.h"
+
+Beg_struct(irt_box)
```

Comment:
Is there an easy way to add the generated docs for the `mesh` table to RTD as an example?

Reply:
It was supposed to be there but it looks like `lua` is not installed on RTD, and you cannot install it, so the build failed. I checked it into the repo and now it should show up.